src/ZeroFootprintSociety/Characters/Teams/TeamGroup.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections.Generic;
+
+namespace ZeroFootPrintSociety.Characters.Teams
+{
+    /// <summary>
+    /// Groups of teams as an enumerator.
```

Comment:
This is an enumeration, not an enumerator.

Reply:
shit you're right haha.